docs/appendices/release-notes/5.9.6.rst

```diff
@@ -52,3 +52,12 @@ Fixes
 
 - Fixed a possible deadlock where concurrent delete requests could take a lock but then never
   release it, leading to requests timing out and threads parked in infinite waits.
+
+- Fixed a regression introduced with :ref:`version_5.6.0` which caused the
```

Comment:
The regression was caused because of a new default in the library right?

Reply:
It was caused by a new version of the library, see https://github.com/crate/crate/issues/17137#issuecomment-2548861318.

Reply:
We could mention this in the note.

Reply:
How will this help?

Reply:
It would just point out the root cause. it's ok like this for me.